### ISSUE TITLE
XDMP-ARGTYPE error thrown when groupToArrays method has only one parameter

### ISSUE THREAD

**Issue by georgeajit** [marklogic/java-client-api#1285]
So we can address your issue, please include the following:
### Version of MarkLogic Java Client API
develop branch (after 5.3.2release)

### Version of MarkLogic Server
10.0-6

### Java version

1.8 u 261

### OS and version

Windows intelliJ IDE

### Input: Some code to illustrate the problem, preferably in a state that can be independently reproduced on our end
```
ModifyPlan plan1 =
            p.fromView("opticFunctionalTest", "detail")
                    .orderBy(p.schemaCol("opticFunctionalTest", "detail", "id"));
    ModifyPlan plan2 =
            p.fromView("opticFunctionalTest", "master")
                    .orderBy(p.schemaCol("opticFunctionalTest", "master" , "id"));
    ModifyPlan plan3 =
            plan1.joinInner(plan2)
                    .where(
                            p.eq(
                                    p.schemaCol("opticFunctionalTest", "master" , "id"),
                                    p.schemaCol("opticFunctionalTest", "detail", "masterId")
                            )
                    )
                    .groupToArrays(p.namedGroupSeq(
                                    p.namedGroup("mId", p.schemaCol("opticFunctionalTest", "master" , "id")),
                                    p.namedGroup("Amt", p.schemaCol("opticFunctionalTest", "detail" , "amount"))
                                )
                    )
                    .orderBy(p.sortKeySeq(p.desc("mId"), p.desc("Amt")));
```

Calling ```groupToArrays(namedGroup); ``` in the test code.

### Actual output: What did you observe? What errors did you see? Can you attach the logs? (Java logs, MarkLogic logs)
```Server Message: XDMP-ARGTYPE: plan.aggregate(null, "", "sample", null, null, null) -- arg1 is not of type String```
### Expected output: What specifically did you expect to happen?
Expected rows.
### Alternatives: What else have you tried, actual/expected?
N/A.

**Comment by ehennum** [comment]
Another case where the root cause here was a server bug (internal number 55817), which is fixed in 10.0-6.1

Please retest with a 10.0-6.1, 10.0-7, or trunk nightly

**Comment by georgeajit** [comment]
Will be tested when server builds are available.
